tvbwidgets/ui/ts_widget.py

```diff
@@ -535,3 +536,21 @@ def _update_fig(self):
             self.fig._redraw(annotations=True)
         except:
             self.fig._redraw(update_data=False)  # needed in case of Unselect all
+
+
+class TimeSeriesBrowserWidget(widgets.VBox, TVBWidgetWithBrowser):
```

Comment:
@liadomide which would be a better name for this widget? 
For the head widget we have: `HeadWidgetBase` for only viewer and `HeadWidget` for file browser + viewer.
Should we use the same rule here?

Reply:
What do you think about: `TimeSeriesWidget` and `TimeSeriesBrowser` ?
Then similarly, the current `HeadWidgetBase` would become `HeadWidget`, and `HeadBrowser`?
So, use the `Widget` suffix only for the simple viewers, but when with file browser, replace the Widget suffix with `Browser`.